src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/FindTableOperation.java

```diff
@@ -35,13 +48,32 @@ public FindTableOperation(
   @Override
   public Uni<Supplier<CommandResult>> execute(
       DataApiRequestInfo dataApiRequestInfo, QueryExecutor queryExecutor) {
-    var cql =
-        "select JSON * from %s.%s limit %s;"
-            .formatted(
-                commandContext.schemaObject().name.keyspace(),
-                commandContext.schemaObject().name.table(),
-                params.limit());
-    var statement = SimpleStatement.newInstance(cql);
+
+    // Start the select
+    Select select =
+        selectFrom(
+                commandContext.schemaObject().tableMetadata.getKeyspace(),
+                commandContext.schemaObject().tableMetadata.getName())
+            .json()
+            .all(); // TODO: this is where we would do the field selection / projection
+
+    // BUG: this probably break order for nested expressions, for now enough to get this tested
```

Comment:
Leave the BUG comment here, 
Seems like here we are not doing any real filters here, POC work.